monitor/content.cpp

```diff
@@ -138,7 +134,7 @@ void Monitor::drawContent()
                             int queueColor = queueDepth < 100 ? 1 : 
                                            queueDepth < 250 ? 6 : 2;
 
-                            string bar = buildProgressBar(queueDepth, kFatQueue, 6);
+                            std::string bar = buildProgressBar(queueDepth, kFatQueue, 6);
```

Comment:
Explicit std:: namespace reference is necessary because we dropped the includes of "main" NDSCPP headers.